assets/styl/_elements/media-video.twig

```diff
@@ -4,10 +4,9 @@
  # @param string   media
  #                 -> html
  # @param array    classes        Additional classes
- # @param string   defaultFormat  [required] Images sizes format ( width x height )
  #}
 {% extends 'media.twig' %}
 
 {% block media_figure %}
-    <div class="media-wrap media-wrap-video">{{ media.html|raw }}</div>
+    <div class="media-wrap media-wrap-video {{ modifier_class }}">{{ media.html|raw }}</div>
```

Comment:
Don't you need to add `modifier_class` in the docbloc above?

Reply:
Every selectors in modifier list in KSS docblock are modifer_class.